### DISCUSSION TITLE
Data validation for SQL connector

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2322] [Help] [answered]
Hi BB Team, 
is there any active data validation for forms using the SQL connector?  I am using conditional formatting to display only parts of the form that are needed, based on the selection of the forms drop-down menu (option picker). I would like to make it so all fields that are displayed with the selection must have an input to be saved is this possible / will it be posable in the future?

**Comment by mjashanks** [comment, marked as answer]
Hey Andrew. We are working on this right now, to be released at the end of the month: https://github.com/Budibase/budibase/issues/2096

**Reply by KernOil** [reply]
Thank you
